src/services/passwordRecovery/hooks/index.js

```diff
@@ -18,6 +21,35 @@ const hashId = (hook) => {
 	}
 };
 
+const sendInfo = (hook) => {
+	if (hook.path === 'passwordRecovery') {
+		hook.app.service('/accounts').get(hook.data.account, {
+			query: {
+				$populate: ['userId'],
+			},
+		}).then((account) => {
+			const recoveryLink = `${process.env.HOST}/pwrecovery/${hook.result._id}`;
+			const mailContent = `Sehr geehrte/r " ${account.userId.firstName} ${account.userId.lastName}, \n\n
+				Bitte setzen Sie Ihr Passwort unter folgendem Link zurück:\n
+				${recoveryLink}\n\n
+				Mit Freundlichen Grüßen\n
+				Ihr ${process.env.SC_SHORT_TITLE || 'Schul-Cloud'} Team`;
+
+			globalHooks.sendEmail(hook, {
+				subject: `Passwort zurücksetzen für die ${process.env.SC_SHORT_TITLE || 'Schul-Cloud'}`,
+				emails: [account.userId.email],
+				content: {
+					text: mailContent,
+				},
+			});
+			return hook;
+		}).catch((err) => {
+			logger.warn(err);
+			throw new NotFound('User Account Not Found');
+		});
+	}
```

Comment:
default case
return context;

Reply:
This part is actually never called since the before hook prevent it already